### PR TITLE
[Coverage/Fix] Disable optimization and remove redundant gcov run

### PR DESCRIPTION
**Pull request by dongju-chae**
This patch disables code optimization for coverage analysis and remove
redundant gcov run.

In fact, we don't need to run the python script, `unittestcoverage.py`
which recursively executes gcov. Because lcov internally executes gcov,
the results of this script are never used.

FYI...
- When program is compiled with coverage option, `.gcno` files are created.
- When such programs are executed, `.gcda` files are created.
- As lcov's input files are `.gcda`, we don't need to run gcov before lcov.

Signed-off-by: Dongju Chae <dongju.chae@samsung.com>

**Self evaluation:**
1. Build test: [*]Passed [ ]Failed []Skipped
2. Run test: [*]Passed [ ]Failed []Skipped

### CODE REVIEW

File: packaging/nnstreamer.spec [hunk 1]
```diff
@@ -484,7 +487,6 @@ popd
 # 'lcov' generates the date format with UTC time zone by default. Let's replace UTC with KST.
 # If you ccan get a root privilege, run ln -sf /usr/share/zoneinfo/Asia/Seoul /etc/localtime
 TZ='Asia/Seoul'; export TZ
-$(pwd)/tests/unittestcoverage.py module $(pwd)/gst $(pwd)/ext %testtarget
```
Comment (jaeyun-jung) [inline packaging/nnstreamer.spec:487]: if unittestcoverage.py is unnecessary, how about removing this file?

Comment (dongju-chae) [reply packaging/nnstreamer.spec:487]: Yes but I am not sure it's okay for me to remove this...

Comment (myungjoo) [reply packaging/nnstreamer.spec:487]: You may remove it if it is not used by .spec file.

Comment (dongju-chae) [reply packaging/nnstreamer.spec:487]: Got it. I will update the PR

**Pull request merged**